packages/contracts/test/voting/token-voting.ts

```diff
@@ -375,47 +405,55 @@ describe('TokenVoting', function () {
     });
 
     it('should not double-count votes by the same address', async () => {
-      await advanceTimeTo(startDate);
+      await advanceIntoVoteTime(startDate, endDate);
 
       await governanceErc20Mock.mock.getPastVotes.returns(1);
 
-      // yes still ends up to be 1 here even after voting
-      // 2 times from the same wallet.
       await voting.vote(id, VoteOption.Yes, false);
       await voting.vote(id, VoteOption.Yes, false);
       expect((await voting.getProposal(id)).yes).to.equal(1);
+      expect((await voting.getProposal(id)).no).to.equal(0);
+      expect((await voting.getProposal(id)).abstain).to.equal(0);
 
-      // yes gets removed, no ends up as 1.
       await voting.vote(id, VoteOption.No, false);
       await voting.vote(id, VoteOption.No, false);
```

Comment:
Why voting twice?

Reply:
The test is called `'should not double-count votes by the same address'`

https://github.com/aragon/core/blob/db90ddf6e54fc65adc5690ebdb8958b46df2d506/packages/contracts/test/voting/token-voting.ts#L407-L429

Reply:
oh sorry :pray: